plugins/scm.js

```diff
@@ -82,6 +83,10 @@ const PARSE_URL = Joi.object().keys({
     token
 }).required();
 
+const CAN_HANDLE_URL = Joi.object().keys({
+    scmUri
```

Comment:
I think `scmContext` can be get by using`getScmContext`.  It may need no parameters..?